### DISCUSSION TITLE
Unable to log in to Assistant PS, Error getting user session

### DISCUSSION THREAD

**Discussion by hkrivellFTCLLC** [EsriPS/arcgis-assistant-feedback#47] [Q&A]
Good Afternoon,

I am attempting to connect to the Assistant.esri-ps.com to our enterprise portal for migration purposes. I was able to connect our destination portal, but our source portal is not connecting (we were able to connect last month). Assistant PS gets to the user approval screen with my user name but once I click approve, the login is attempted then it defaults to the destination portal, unable to connect to the source portal. I reviewed the Dev console and found the following error during Get User Session (Complete0Auth) .  This returns an error 401 and does not log into our source portal.

![image](https://user-images.githubusercontent.com/64435508/172883398-cf487950-b46d-4df0-926d-253700b308ec.png)

I'm not sure how this is resolved. Has anyone been able to fix this or know why complete0auth  is failing?

**Comment by nheminger** [comment]
Hello @hkrivellFTCLLC,

I'm taking a look at this issue. Do you see any response content from the 401 response? What is the path of the request above (excluding your server name and any token parameters)?

Thanks!

**Comment by slibby** [comment]
See https://github.com/EsriPS/arcgis-assistant-feedback/issues/48 for a likely resolution. Sorry for the delay on this!
